earthfile2llb/interpreter.go

```diff
@@ -600,6 +650,7 @@ func (i *Interpreter) handleCopy(ctx context.Context, cmd spec.Command) error {
 	keepTs := fs.Bool("keep-ts", false, "Keep created time file timestamps")
 	keepOwn := fs.Bool("keep-own", false, "Keep owner info")
 	ifExists := fs.Bool("if-exists", false, "Do not fail if the artifact does not exist")
+	allowPrivilegedFlag := fs.Bool("allow-privileged", false, "Allow targets to assume privileged mode")
```

Comment:
I wonder if this works as a workaround for not supporting short-form flags?

```suggestion
	allowPrivilegedFlag1 := fs.Bool("allow-privileged", false, "Allow targets to assume privileged mode")
	allowPrivilegedFlag2 := fs.Bool("P", false, "Allow targets to assume privileged mode")
	allowPrivilegedFlag := allowPrivilegedFlag1 || allowPrivilegedFlag2
```

Reply:
I was thinking about that, but figure it's best to just swap out the lib. It would be a bit funny to allow `-allow-privileged` and `--P` if we did this change.

Reply:
Yeah, I agree we should swap out the lib eventually. I was thinking more like a stop-gap solution - despite the weird `--P` 😄